types/types.go

```diff
@@ -176,6 +176,7 @@ const (
 	LonghornLabelVersion                    = "version"
 	LonghornLabelAdmissionWebhook           = "admission-webhook"
 	LonghornLabelConversionWebhook          = "conversion-webhook"
+	LonghornLabelManagedStorageClass        = "managed-storage-class"
```

Comment:
Why introduce this?

Reply:
As Phan mentioned, https://github.com/longhorn/longhorn/issues/9391#issuecomment-2332466349